common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -923,47 +901,62 @@ div.problem {
 // +Problem - Actions
 // ====================
 div.problem .action {
-  margin-top: $baseline;
+  @include margin($baseline 0);
+  min-height: $baseline;
 
-  .save, .check, .show, .reset, .hint-button {
-    @include margin-right($baseline/2);
-    margin-bottom: ($baseline/2);
-    height: ($baseline*2);
-    vertical-align: middle;
-    text-transform: uppercase;
-    font-weight: 600;
-    @media print {
-      display: none;
+  .problem-action-buttons-wrapper {
+    margin-bottom: $baseline / 2;
+
+    @media (min-width: $bp-screen-md) {
+      @include right($baseline * 1.5);
+      margin-top: -$baseline / 2;
+      position: absolute;
     }
-  }
 
-  .save {
-    @extend .blue-button !optional;
   }
 
-  .show {
+  .problem-action-button-wrapper {
+    display: inline-block;
+    @include border-right(1px solid $light-gray1);
```

Comment:
@Includes go at the top of their block (variables, includes, extends, properties)

https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order